games-emulation/higan/metadata.xml

```diff
@@ -7,6 +7,10 @@
 	</maintainer>
 	<use>
 		<flag name="icarus">Build icarus library manager</flag>
-		<flag name="udev">Enable udev based input</flag>
```

Comment:
Is it intentional to remove this? I'm asking since this still holds true.

Reply:
This local description is essentially the same as the global description for the udev USE flag, so I'm removing it as redundant.

Reply:
Ah, OK, that makes sense!